modules/ROOT/pages/dashboard-custom-config.adoc

```diff
@@ -61,18 +61,20 @@ image::dashboard-custom-config-high-level.png[Dashboard Configuration]
 | Tags | A short identifier that can help you locate a dashboard in the custom dashboard list. See xref:dashboards-using.adoc#favorite_dashboard[Setting Favorite Custom Dashboards] for an example.
 | Timezone a|
 
-Most graphs track data over time, so your time options are important. Here are your options:
+Most graphs track data over time, so your time options are important. Your options are:
```

Comment:
no "your" before time options? Makes sentence harder to read? And Time options instead of Your options?